shared/devices/device-page/index.desktop.js

```diff
@@ -62,41 +62,32 @@ const Timeline = ({timeline}) => (
   </Box>
 )
 
-const Render = ({
-  name,
-  type,
-  deviceID,
-  currentDevice,
-  timeline,
-  revokedAt,
-  showRevokeDevicePage,
-  onBack,
-  bannerBackgroundColor,
-  bannerColor,
-  bannerDesc,
-  icon,
-  revokeName,
-}: Props) => (
+const Render = (props: Props) => (
```

Comment:
this top level type spread doesn't really give us much, just adds a bunch of lines